specification/schemas/ReturnReason.json

```diff
@@ -0,0 +1,45 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/ReturnReasonResource"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "description": {
+              "type": "string",
+              "example": "The item isn't what I expected it to be."
+            },
+            "tags": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "example": "shoes"
+              }
+            },
+            "accepts_attachments": {
+              "type": "boolean",
+              "example": true
+            },
+            "requires_attachments": {
+              "type": "boolean",
+              "example": true
+            }
+          }
+        },
+        "relationships": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "shop": {
+              "$ref": "#/components/schemas/ShopRelationship"
+            }
+          }
+        }
```

Comment:
ReturnReasons are generic things that our customers (shops) can decide to activate/use. One ReturnReason can be used by 0 or more shops. Only the linked ReturnReasons will be returned by the shop-specific endpoint.

If we would expose any relationship on a ReturnReason, I think it should be an `owner` relationship with us (the broker) as owner. But I don't think this is needed yet.

Reply:
I think we should 🔥 this shop relationship